client/mainmenu/CMainMenu.cpp

```diff
@@ -88,6 +88,18 @@ CMenuScreen::CMenuScreen(const JsonNode & configNode)
 
 	pos = background->center();
 
+	const auto& logoConfig = config["logo"];
+	if (!logoConfig.isNull() && logoConfig["name"].isVector() && !logoConfig["name"].Vector().empty()) {
```

Comment:
- place opening bracket on new line
- there is no need to check json node type. Accessing json using wrong type is well-defined behavior and will return default-constructed value, in this case - empty vector
```suggestion
	if (!logoConfig["name"].Vector().empty()) 
	{
```

Reply:
Simplified code in latest commit